CONTRIBUTING.md

```diff
@@ -19,31 +19,31 @@ _If you are new to git and GitHub, check out [this Hello World guide](https://gu
 Here, we include a _typical_ workflow for filing a pull request to contribute to this project.
 
 1. Fork this repository to your own account: https://help.github.com/en/articles/fork-a-repo
-2. Treat `AlexsLemonade/OpenPBTA-analysis` as an upstream repository. 
+2. Treat `PediatricOpenTargets/OpenPedCan-analysis` as an upstream repository.
 You should follow these instructions for configuring a remote that points to an upstream repository here: https://help.github.com/en/articles/configuring-a-remote-for-a-fork
-3. When you want to make changes and ultimately file a pull request against the `master` branch of this repository, you can start by making a branch from the `master` branch of your fork named for the changes you intend to make. 
+3. When you want to make changes and ultimately file a pull request against the `dev` branch of this repository, you can start by making a branch from the `dev` branch of your fork named for the changes you intend to make. 
 For example, you could name your branch `gene-expression-clustering` 
 Then, [check out the new branch](https://gist.github.com/markSci5/5916003) (e.g., `gene-expression-clustering`).
 Commit and push your changes to this branch. 
 You will then be able to file a pull request from this branch: https://help.github.com/en/articles/creating-a-pull-request-from-a-fork
-4. Once your pull request is approved and merged by the project maintainers, it's time to keep the `master` branch of your fork up-to-date with the `master` branch of this repository by following this process: https://help.github.com/en/articles/syncing-a-fork.
+4. Once your pull request is approved and merged by the project maintainers, it's time to keep the `dev` branch of your fork up-to-date with the `dev` branch of this repository by following this process: https://help.github.com/en/articles/syncing-a-fork.
 Here's a typical series of steps for performing that:
 
 ```
-# fetch the changes from the AlexsLemonade repository
+# fetch the changes from the PediatricOpenTargets/OpenPedCan-analysis repository
 git fetch upstream
 
-# checkout your own master branch
-git checkout master
+# checkout your own dev branch
+git checkout dev
 
-# merge the upstream/master branch into your own master branch
-git merge upstream/master
+# merge the upstream/dev branch into your own dev branch
+git merge upstream/dev
 
-# push the changes to your master branch to the remote repository
+# push the changes to your dev branch to the remote repository
 git push
 ```
 
-You do not want to commit changes to your `master` branch, any changes to your `master` branch should come from the upstream `master` branch.
+You do not want to commit changes to your `dev` branch, any changes to your `dev` branch should come from the upstream `dev` branch.
 
 ### Size and composition of pull requests
```

Comment:
Can you also add a recommended size of new PR as 400 lines of code, and if the PR will exceed, to break it down into smaller PRs - can refer to "stacked PRs" below.

Reply:
Sure. Will do.